src/NuGet.Core/NuGet.PackageManagement/BuildIntegration/DependencyGraphRestoreUtility.cs

```diff
@@ -51,14 +51,16 @@ public static class DependencyGraphRestoreUtility
                 forceRestore,
                 isRestoreOriginalAction,
                 additionalMessages: null,
+                progressReporter: null,
```

Comment:
Should this method also have the obsolete attribute?  Or do we intend on "supporting" this API?

Reply:
There are 40 tests calling this. 

We can obsolete and replace though. 

If you think we should do that, I can create a PR and make it call other methods instead.

Reply:
It just seems weird to me that we mark the other method as obsolete, but not this one. We shouldn't "support" an API just because a test uses it, but a test using it and it being painful to change is an indication that it might be painful for our SDK customers to change as well. THerefore normally we should reconsider whether or not to obsolete the other API, but thi is the `NuGet.PackageManagement` project which I care less about.

Reply:
The biggest reason why that method is obsolete is because it has unused parameters. 

In particular, the solution manager is not used, so I just wanted us to get rid of methods with unused parameters.

Reply:
Ok, cool. I didn't notice that. That is a great reason to obsolete a method!